### PR TITLE
Rename FV::Div_a_Grad_perp to FV::Div_a_Grad_perp

### PR DESCRIPTION
**Pull request by dschwoerer**
This operator is calculating the Divergence of the Gradient (i.e.
Laplace), not the Divergence of the Laplacian.

Should this be called `Div_perp_a_Grad_perp` or even `Div_perp_K_Grad_perp` (similar to the parallel version)?

### CODE REVIEW

**Review by bendudson — APPROVED** [review]
Thanks @dschwoerer I agree the new name is better

File: include/bout/fv_ops.hxx [hunk 1]
```diff
@@ -10,14 +10,20 @@
...
 namespace FV {
-  /*!
-   * Div ( a Laplace_perp(x) )  -- Vorticity
-   */
-  const Field3D Div_a_Laplace_perp(const Field3D &a, const Field3D &x);
-  
+/*!
+ * Div ( a Grad_perp(f) ) -- ∇⊥ ( a ⋅ ∇⊥ f) -- Vorticity
+ */
+const Field3D Div_a_Grad_perp(const Field3D& a, const Field3D& x);
```
Comment (ZedThree) [inline include/bout/fv_ops.hxx:20]: If we're touching this, let's drop `const` at the same time:

```suggestion
Field3D Div_a_Grad_perp(const Field3D& a, const Field3D& x);
```

File: include/bout/fv_ops.hxx [hunk 2]
```diff
@@ -10,14 +10,20 @@
...
+
+[[deprecated("Please use Div_a_Grad_perp instead")]] inline const Field3D
```
Comment (ZedThree) [inline include/bout/fv_ops.hxx:22]:
```suggestion
[[deprecated("Please use Div_a_Grad_perp instead")]] inline Field3D
```

File: src/mesh/fv_ops.cxx [hunk 3]
```diff
@@ -8,224 +8,222 @@
 
 namespace FV {
 
-  // Div ( a Laplace_perp(f) )  -- Vorticity
-  const Field3D Div_a_Laplace_perp(const Field3D &a, const Field3D &f) {
-    ASSERT2(a.getLocation() == f.getLocation());
+// Div ( a Grad_perp(f) ) -- ∇⊥ ( a ⋅ ∇⊥ f) --  Vorticity
+const Field3D Div_a_Grad_perp(const Field3D& a, const Field3D& f) {
```
Comment (ZedThree) [inline src/mesh/fv_ops.cxx:12]:
```suggestion
Field3D Div_a_Grad_perp(const Field3D& a, const Field3D& f) {
```

**Review by ZedThree — CHANGES_REQUESTED** [review]
LGTM, thanks for the nice deprecation notice too!

**Review by ZedThree — APPROVED** [review]

**Comment by ZedThree** [comment]
Failing test is clang-tidy-review which is fixed in both master/next

**Comment by dschwoerer** [comment]
Does the title contain `FV::Div_a_Grad_perp` twice? Old name was `FV::Div_a_Laplace_perp` - sorry for not providing a better title.

**Comment by ZedThree** [comment]
Oops, sorry, I clearly didn't read things properly! I'll just have to remember to fix it for the changelog

**Pull request merged**